extensions/wikia/ArticleVideo/bower_components/html5-skin/tests/components/moreOptionsPanel-test.js

```diff
@@ -0,0 +1,123 @@
+jest.dontMock('../../js/components/moreOptionsPanel')
+    .dontMock('../../js/components/utils')
+    .dontMock('../../js/components/icon')
+    .dontMock('../../js/constants/constants')
+    .dontMock('../../js/mixins/animateMixin')
+    .dontMock('classnames');
+
+var React = require('react');
+var TestUtils = require('react-addons-test-utils');
+var CONSTANTS = require('../../js/constants/constants');
+var MoreOptionsPanel = require('../../js/components/moreOptionsPanel');
+var skinConfig = require('../../config/skin.json');
+var Utils = require('../../js/components/utils');
+
+// start unit test
+describe('MoreOptionsPanel', function () {
+  it('creates more options panel', function () {
+
+    var oneButtonSkinConfig = Utils.clone(skinConfig);
+    oneButtonSkinConfig.buttons.desktopContent = [
+      {"name":"share", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":200 },
+      {"name":"discovery", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":200 },
+      {"name":"closedCaption", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":200 },
+      {"name":"quality", "location":"controlBar", "whenDoesNotFit":"moveToMoreOptions", "minWidth":200 }
+    ];
+
+    var mockController = {
+      state: {
+        isMobile: false,
+        volumeState: {
+          volume: 1
+        },
+        closedCaptionOptions: {availableLanguages: true},
+        videoQualityOptions: {
+          availableBitrates: true
+        },
+        discoveryData: true,
+        moreOptionsItems: oneButtonSkinConfig.buttons.desktopContent
+      },
+      toggleDiscoveryScreen: function() {
+        discoveryScreenToggled = true;
+      },
+      toggleShareScreen: function() {
+        shareScreenToggled = true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "shareScreenToggled" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)